modules/ROOT/pages/_partials/nav-dw.adoc

```diff
@@ -133,34 +156,62 @@
 **** xref:dw-objects-functions-someentry.adoc[someEntry]
 **** xref:dw-objects-functions-takewhile.adoc[takeWhile]
 **** xref:dw-objects-functions-valueset.adoc[valueSet]
+*** xref:dw-periods.adoc[Periods (dw::core::Periods)]
+**** xref:dw-periods-functions-between.adoc[between]
+**** xref:dw-periods-functions-days.adoc[days]
```

Comment:
days:

TODO: parameter name "nDays". Do we want to use that convention, i.e., "n[X]" to mean number of some unit?

Reply:
PR with edits: https://github.com/mulesoft/data-weave/pull/1295/files

Reply:
added to list of param names to review